utils/python/CIME/case_run.py

```diff
@@ -393,17 +209,21 @@ def case_run(case):
     lid = time.strftime("%y%m%d-%H%M%S")
     os.environ["LID"] = lid
 
-    save_timing_setup_acme(case, lid)
+    save_prerun_provenance(case)
 
     for _ in range(data_assimilation_cycles):
```

Comment:
One question I had here, if data_assimilation_cycles is more than 1, doesn't that break all our timing savings? The LID is the same regardless of how many times this loop runs, so wouldn't the timing data get overwritten?